cmd/format-xl.go

```diff
@@ -868,22 +727,19 @@ func initFormatXL(storageDisks []StorageAPI) (err error) {
 	var jbod = make([]string, len(storageDisks))
 
 	// Initialize formats.
-	var formats = make([]*formatConfigV1, len(storageDisks))
+	var formats = make([]*formatXLV1, len(storageDisks))
 
 	// Initialize `format.json`.
 	for index, disk := range storageDisks {
 		if disk == nil {
 			continue
 		}
-		// Allocate format config.
```

Comment:
what was wrong in this? how about writing a newFormatXLV1() function?

Reply:
```
                // Allocate format config.
                formats[index] = &formatConfigV1{
                        Version: formatFileV1,
                        Format:  formatBackendXL,
                        XL: &xlFormat{
                                Version: xlFormatBackendV1,
                                Disk:    mustGetUUID(),
                        },
                }
```

the comment `// Allocate format config.` looked redundant as the code is obvious.

Reply:
will add newFormatXLV1()